sbin/pfmon

```diff
@@ -391,7 +411,7 @@ run all runtasks
 sub runtasks {
     my $mask = POSIX::SigSet->new(POSIX::SIGCHLD());
     sigprocmask(SIG_BLOCK,$mask);
-    while(@REGISTERED_TASKS) {
+    while(@REGISTERED_TASKS && $process) {
```

Comment:
Not touching @REGISTERED_TASKS :P
Yep everything is spawning back as supposed !